redis_hashring/__init__.py

```diff
@@ -345,8 +342,10 @@ def gevent_start(self):
         Helper method to start the node for gevent-based applications.
         """
         import gevent
+        import gevent.select
 
-        self._poller_greenlet = gevent.spawn(self.poll, as_greenlet=True)
+        self._poller_greenlet = gevent.spawn(self.poll)
```

Comment:
Swap the order of this line and the next. Likely not an issue, but we want to make sure `self._select` is set when we spawn.

Reply:
Done in https://github.com/closeio/redis-hashring/pull/16/commits/7f2e1e78b33f4ee4dd5e1686611470be72727497.